src/components/Popups/ReplyField/MentionItem.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { ContentState } from 'draft-js';
+import './MentionItem.scss';
+
+type Props = {
+    children: React.ReactNode;
+    contentState: ContentState;
+    decoratedText: string;
+    entityKey?: string;
+};
+
+const MentionItem = ({ contentState, entityKey, children }: Props): JSX.Element => {
+    const id = entityKey ? contentState.getEntity(entityKey).getData().id : '';
+
+    return (
+        <a className="ba-MentionItem-link" href={`/profile/${id}`}>
```

Comment:
I think @mickr might have left this comment, but should we render a different element if `id` is indeed an empty string?

Reply:
> I think @mickr might have left this comment, but should we render a different element if `id` is indeed an empty string?

Good idea. `id` may be undefined even though `entityKey` won't.